ddtrace/internal/sampling.py

```diff
@@ -166,3 +165,8 @@ def apply_span_sampling_tags(self, span):
         span.set_metric(_SINGLE_SPAN_SAMPLING_RATE, self._sample_rate)
         if self._max_per_second:
             span.set_metric(_SINGLE_SPAN_SAMPLING_MAX_PER_SEC, self._max_per_second)
+
+
+def is_single_span_sampled(span):
```

Comment:
Should we move this method to the module where it's used? `is_single_span_sampled` is only used in `ddtrace/internal/processor/trace.py`.

Reply:
That seems like it would make sense to me, but I'd like to know what @brettlangdon  or @Kyle-Verhoog  think?

Reply:
this helper is fine where it is. since it is using the private constant `_SINGLE_SPAN_SAMPLING_MEHCANISM` it makes sense to me to be here.